cogs/meta.py

```diff
@@ -250,27 +253,61 @@ async def support(self, ctx):
 
 		await ctx.send(self.bot.support_link)
 
-	@commands.command(aliases=['source'])
-	async def code(self, ctx, *, command: str = None):
-		'''Show command code or get GitHub repo link.'''
+	@commands.command(name="source", aliases=["src", "code"])
+	async def source(self, ctx: commands.Context, source_item: str = None):
```

Comment:
Changing command names is usually an unpopular move among users, keep old name of `code`

Reply:
Also remove aliases

Reply:
I changed the name back to `code` but kept the aliases since `source` is a current alias.